hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/AbstractCSQueue.java

```diff
@@ -176,7 +174,7 @@ protected void setupConfigurableCapacities() {
 
   protected void setupConfigurableCapacities(
       CapacitySchedulerConfiguration configuration) {
-    CSQueueUtils.loadCapacitiesByLabelsFromConf(getQueuePath(), queueCapacities,
+    CSQueueUtils.loadCapacitiesByLabelsFromConf(getQueuePathObject(), queueCapacities,
```

Comment:
No need to use getQueuePathObject getter inside this class, you can use the field directly.
This applies to all getQueuePathObject() calls from this class

Reply:
I don't see all of these fixed in this class. Anyway, let's go with the current version.

Reply:
OK, modified all the usages. Will commit with the updated version.